src/main/java/mate/academy/dao/OrderDao.java

```diff
@@ -0,0 +1,19 @@
+package mate.academy.dao;
+
+import java.util.List;
+import java.util.Optional;
+import mate.academy.model.Order;
+import mate.academy.model.ShoppingCart;
+
+public interface OrderDao {
+
+    Order completeOrder(ShoppingCart shoppingCart);
```

Comment:
Let's create Create and Update methods instead of Complete.
This logic (Complete) should be placed into the Service package.

Reply:
I agree with Daria) Let's add methods create() and update() and remove completeOrder() from dao layer